go.mod

```
@@ -4,67 +4,79 @@ go 1.19
 
 replace (
 	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20190514055449-b30ab9de040e
-	k8s.io/client-go => k8s.io/client-go v0.21.0
+	k8s.io/client-go => k8s.io/client-go v0.25.12
 )
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/rancher/rke v1.3.0-rc8.0.20210706205346-22b82828ffa0
-	github.com/sirupsen/logrus v1.7.0
-	github.com/stretchr/testify v1.7.0
-	golang.org/x/mod v0.3.1-0.20200828183125-ce943fd02449
-	k8s.io/apimachinery v0.21.0
-	sigs.k8s.io/yaml v1.2.0
+	github.com/rancher/rke v1.4.7
```

The v2.8 branch is used in Rancher v2.8 and REK 1.5, so we can bump rke to a v1.5 version
v1.5.0-rc8 is the latest we have
```suggestion
	github.com/rancher/rke v1.5.0-rc8
```